packages/react-components/src/components/VideoEffects/VideoBackgroundEffectsPicker.tsx

```diff
@@ -136,7 +136,9 @@ export const _VideoBackgroundEffectsPicker = (props: _VideoBackgroundEffectsPick
           data-ui-id="video-effects-picker-row"
         >
           {options.map((option) => (
-            <_VideoEffectsItem {...option} key={option.itemKey} itemKey={option.itemKey} />
+            <FocusZone key={option.itemKey} shouldFocusOnMount={!!(option.itemKey === 'none')}>
```

Comment:
mega nit: can 
```
shouldFocusOnMount={!!(option.itemKey === 'none')}
```
be simplified to
```
shouldFocusOnMount={option.itemKey === 'none'}
```

Reply:
Not sure why do we need this condition here `shouldFocusOnMount={!!(option.itemKey === 'none')`

Reply:
I tried this the other way and it did not work. the !! makes the check truthy like an if statement. Typescript was throwing a compiler error with out it 
![image](https://github.com/Azure/communication-ui-library/assets/94866715/60b2fdf6-133c-4ddb-9383-85800aa5a832)

Reply:
yeah, there can be a case if the active effect is updated after creating the adapter, `none` would be the first item but not the active effect